scripts/eft-story-generate.py

```
@@ -0,0 +1,176 @@
+
+    The reference is wrapped in a nested envelope; unwrap to the quest array,
+    tolerating either the enveloped shape or a already-unwrapped `{data: [...]}`.
+    """
+    raw = json.loads(REF.read_text())
+    node = raw.get("response", raw)
+    node = node.get("decoded_response", node)
+    data = node.get("data", node)
+    return data
+
+
+def main():
```

Ruff (0.15.20)

[warning] Too many statements (55 > 50)

(PLR0915)